README.md

```diff
@@ -22,17 +22,18 @@ cd ../..
 scramv1 b -j 16 
 ```
 
-## Submit the limit jobs on lxplus
+## Submit the limit jobs
 example for GGH only (for VBF few updates have still to be done by hand in the code)
 ```bash
-UserCode/llvv_fwk/test/hzz2l2v/computeLimit
+if [[ $HOSTNAME =~ "iihe" ]]; then yes | scp lxplus.cern.ch:/eos/cms/store/user/hbrun/analysis/plotters/plotter_2017_06_08_forLimits_JulyFromAlessio.root $CMSSW_BASE/src/UserCode/llvv_fwk/test/hzz2l2v/computeLimit/.; fi
```

Comment:
Maybe that would be more efficient to have a local copy of the plotter on the IIHE T2 no ?

Reply:
Why not, a suggestion of where?

Reply:
I could copy the plotter in my directory and they you update the compute limit to handle it in case of IIHE ?

Reply:
Here: 
`/storage_mnt/storage/user/hbrun/public/plotters/plotter_2017_06_08_forLimits_JulyFromAlessio.root`